docs/source/guides/feature_descriptions.ipynb

```diff
@@ -152,9 +152,7 @@
    "source": [
     ".. note::\n",
     "\n",
-    "    When setting a description on a column in a DataFrame as described above, you will have to go through the ``columns`` attribute of the Woodwork DataFrame, ``df.ww.columns[col_name].description``, in order to set the column's description in a way that will be propogated to the feature description. \n",
-    "    \n",
-    "    The other option for selecting the series from the DataFrame with ``df.ww[col_name].ww.description`` will create an entirely new Woodwork Series that is not related to the EntitySet from which feature descriptions are built."
+    "    When setting a description on a column in a DataFrame as described above, be careful to avoid setting the description via ``df.ww[col_name].ww.description``. The use of ``df.ww[col_name]`` creates an entirely new Series object that is not related to the EntitySet from which feature descriptions are built. Therefore, setting the description in any way other than going through the ``columns`` attribute will not set the column's description in a way that will be propogated to the feature description. "
```

Comment:
I like this a lot.

We could also add on at the end "This also applies when updating column metadata."  Not necessary for the note but might be useful to share

Reply:
I'm thinking this gotcha might be an important thing to reference in the Woodwork typing guide. Since, like you said, it'll be relevant to updating information stored in he metadata as well.

There isn't really a section in the guide on accessing descriptions and metadata, but it might be worth it to add a section on "Updating a Columns Typing Information" at the bottom

Reply:
I'm thinking that adding the situation here to the FAQs in #1578 will mean that we shouldn't need to reference metadata here. I think it's still useful to have a note specific to the situation at hand in these guides when this situation comes up but that we don't need to reference the other situations.